### PR TITLE
Change Endangered Acreage Label for facilities form. Fix Tadjik Marko…

### PR DESCRIPTION
**Pull request by jcavena**
…ur species name.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 Species.where(name: 'Nubian ibex').first_or_create
 Species.where(name: 'Scimitar-horned Oryx').first_or_create
 Species.where(name: 'Sable Antelope').first_or_create
-Species.where(name: 'Tajik Markhor').first_or_create
+Species.where(name: 'Tadjik Markhor').first_or_create
```
Comment (jcavena) [inline db/seeds.rb:14]: Nope.

Comment (pachacamac) [inline db/seeds.rb:14]: Yea let's add

```
Style/StringLiterals:
  EnforcedStyle: false
```

Also :+1:

Comment (jcavena) [inline db/seeds.rb:14]: It seems we use the single_quote style. Maybe we switch from double_quote to single_quote instead and see if that makes less noise before disabling?

Comment (pachacamac) [inline db/seeds.rb:14]: I usually go for single quotes unless I have variables in there. The only thing I really don't like is something like `'foo' + x + 'bar'`

Comment (jcavena) [inline db/seeds.rb:14]: Same. I use double quotes when I need string interpolation.

**Pull request merged**